EIPS/eip-6932.md

```diff
@@ -0,0 +1,109 @@
+
+- Flexibility: The Subscription Token framework accommodates various subscription models, such as monthly, quarterly, or annual billing cycles. Additionally, merchants have the option to define trial periods, upgrade/downgrade plans, and cancellation policies, providing a versatile foundation for a wide range of subscription-based businesses.
+
+- Security: The Subscription Token employs established security measures, including the use of cryptographic signatures, to ensure the integrity and authenticity of subscription-related transactions. This protects both users and merchants from unauthorized access and potential malicious activities.
+
+### SubscriptionToken
+
+#### subscribers
+
+subscribers list for the subscription
+
+#### subscriptionInfo
+
+struct containing metadata of the subscription, ex, `subscriptionID`, `subscriptionName`, `subscriptionDesc` and `subscriptionTandC`, and many more fields if reqd.
+
+#### subscriptionFee
+
+the fee amount for the subcription
+
+#### subscriptionFrequency
+
+frequency of subscription, interval at which the `subscriptionFee` will be charged.
+
+#### subscribe
+
+function to subscribe toh the subscription
+
+#### unsubscribe
+
+revoke subscription
+
+```solidity
+interface ISubscriptionERC20 {
+  /// @dev map subscribers address, returns address(0) if `idx` is not found
+  /// @param idx: the index of the map values
+  /// @return the address at index `idx` of subscribers
+  function subscribers(uint idx) external view returns (address);
+
+  /// @dev can be extended accorging to the requirements
+  /// @return subscriptionID, and metadata payload provided, may vary
+  function subscriptionInfo() external view returns ( uint, string memory, string memory, string memory );
+
+  /// @dev subscribes to the subscription, can be payable
+  function subscribe() external;
+
+  /// @dev unsubscribe the subscription
+  function unsubscribe() external;
+
+  /// @dev view or pure can be used
+  /// @return the subscription fee
+  function subscriptionFee() external view returns (uint256);
+
+  /// @dev view or pure can be used
+  /// @return get the subscription frequency
+  function subscriptionFrequency() external view returns (uint);
+}
+```
+
+## Rationale
+
+The introduction of the Subscription Token EIP addresses several critical aspects of the Ethereum ecosystem, providing a rationale for its adoption and implementation.
+
+Traditional subscription models often require users to manually initiate recurring payments, leading to inconvenience and potential disruptions in service delivery. By establishing a Subscription Token standard, users can authorize automatic deductions from their balances at predefined intervals. This automation simplifies the subscription management process, ensuring uninterrupted access to subscribed services without the need for repetitive manual actions.
```

Comment:
This all reads more like it belongs in the Motivation section. The Rationale section should justify technical choices made within the EIP itself, while the Motivation justifies the EIP as a whole.